src/app/workspace/resource/properties/properties.component.html

```diff
@@ -146,6 +146,36 @@ <h3 class="label mat-title">
                 </div>
             </div>
         </div>
+
+        <div *ngIf="incomingLinkResources.length > 0">
+            <div class="property border-top">
+                <div class="property-label">
+                    <!-- label of the incoming links -->
+                    <h3 class="label mat-subheading-1">
+                        has incoming link
+                    </h3>
+                </div>
+                <div class="property-value">
+                    <div *ngIf="loading">
```

Comment:
You should use the `dsp-progress-indicator` in case of loading

Reply:
I think right now its not in the dsp-app

Reply:
you can either import it from dsp-ui or leave a comment to replace it with the progress indicator once it's been migrated to the app